src/tensorflow.cc

```diff
@@ -1692,8 +1696,11 @@ ModelState::AutoCompleteConfig()
     // Attempt to auto-complete the config with first loaded model file.
     // 'default_model_filename' is the first model file to try.
     std::string default_model_filename;
-    ModelConfig().MemberAsString(
-        "default_model_filename", &default_model_filename);
+    if (ModelConfig().Find("default_model_filename")) {
+      RETURN_IF_ERROR(ModelConfig().MemberAsString(
+          "default_model_filename", &default_model_filename));
+    }
```

Comment:
Did this ever throw errors before the if statement was added?

Reply:
I think we can remove the if statement here as `default_model_filename` field exists in the model config no matter if it is set or not. Thanks for the catch!